tpp-app/src/home/pages/LogMultipleDatesScreen.js

```diff
@@ -1,43 +1,276 @@
-import React from 'react';
-import { View, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { View, StyleSheet, Text, TouchableOpacity, Alert } from 'react-native';
 import CloseIcon from '../../../ios/tppapp/Images.xcassets/icons/close_icon.svg'
+import { CalendarList } from 'react-native-calendars';
+import { STACK_SCREENS } from '../CalendarNavigator';
+import { getCalendarByYear, getSymptomsFromCalendar, initializeEmptyYear } from '../../services/utils/helpers';
+import { LogMultipleDayPeriod } from '../../services/LogSymptomsService';
+import SubmitIcon from '../../../ios/tppapp/Images.xcassets/icons/checkmark';
+import SelectedIcon from '../../../ios/tppapp/Images.xcassets/icons/selected_icon';
+
+const DayComponent = ({props}) => {
+    const {onPress, date, marking} = props;
+    // console.log(selectedDates);
+    
+    // const [backgroundColor, setBackgroundColor] = useState("#FFFFFF")
```

Comment:
suggest remove big commented out blocks  if not using

Reply:
true true, removed!